### DISCUSSION TITLE
Not able to execute JS script with seleniumbase

### DISCUSSION THREAD

**Discussion by S2001B** [seleniumbase/SeleniumBase#2623] [General]
I don't know how to execute JS code with seleniumbase (the equivalent of executing commands in the browser console of a webpage). I tried a couple of different approaches, but did not succeed. Bellow is an example code snippet I tried, but It returns None value for some reason.

```python
from seleniumbase import SB

WEBSITE_URL= "link"

with SB(uc=True, headed=True) as sb:
   sb.driver.get(WEBSITE_URL)
   a = sb.execute_script("function getStaticValue() {return '42';} getStaticValue()")
   print(a)
   sb.sleep(10)
```
Could you please explain how to correctly execute JS with seleniumbase, and if possible, direct me in the documentation.

**Comment by mdmintz** [comment]
You need the `return` to get the value from it:

```python
sb.execute_script("function getStaticValue() {return '42';} return getStaticValue()")
```
